test/common.py

```diff
@@ -407,8 +407,7 @@ def metafunc(self, fs, *args, **kwargs):
       assert fs is None
     func(self, *args, **kwargs)
 
-  parameterize(metafunc, {'': (None,),
-                          'nodefs': ('nodefs',)})
+  parameterize(metafunc, {'': (None,), 'nodefs': ('nodefs',)})
```

Comment:
All of these uses of `parameterize` are deliberately spread so that there is one test case per line.   Its important for readability.

Reply:
See the `format suppression` comment in the commit message.

Reply:
Adding a trailing comma after the last dict item will cause `ruff format` to make things clear.
```suggestion
  parameterize(
    metafunc,
    {
      '': (None,),
      'nodefs': ('nodefs',),
    },
  )

```